UnityProject/Assets/Scripts/Systems/Antagonists/Antags/Blob/BlobStarter.cs

```diff
@@ -83,7 +83,12 @@ private void Start()
 
 		private void OnEnable()
 		{
+#if UNITY_EDITOR
+			UpdateManager.Add(PeriodicUpdate, 0.15f);
```

Comment:
might be better just to have a debug bool on it, that sskips the updateTime stuff?

Reply:
no need